lib/puppet/type/sensu_plugin.rb

```diff
@@ -0,0 +1,82 @@
+require_relative '../../puppet_x/sensu/type'
+require_relative '../../puppet_x/sensu/array_property'
+require_relative '../../puppet_x/sensu/hash_property'
+require_relative '../../puppet_x/sensu/integer_property'
+
+Puppet::Type.newtype(:sensu_plugin) do
+  desc <<-DESC
+@summary Manages Sensu plugins
+@example Install a sensu plugin
+  sensu_plugin { 'disk-checks':
+    ensure  => 'present',
+  }
+
+@example Install latest version of a sensu plugin
+  sensu_plugin { 'disk-checks':
+    ensure  => 'present',
+    version => 'latest',
```

Comment:
example should show how to pin to a specific version as sensu recommends that you pin your check versions

Reply:
Added example of passing specific version.